transformation/src/api/amqp/pipelineConfigTriggerRequestAmpq.ts

```diff
@@ -0,0 +1,39 @@
+import { isObject, isNumber, isString, hasProperty } from '../../validators'
+
+export interface PipelineConfigTriggerRequestAmpq {
+  datasourceId: number;
+
+  data: string;
+  dataLocation: string;
```

Comment:
```suggestion
```
This interface should match exactly the adapter's `DatasourceImportedEvent.java` class, which contains the structure of the event during creation.

Reply:
The interface was created this way to enable both thin and fat events. Personally, I agree with you that we should keep the model as simple and possible and add extra fields only if needed, but I would like to hear @georg-schwarz opinion before changing this since we have been discussing the issue before.

Reply:
I missed mentioning in my comment, that the `dataLocation` property is also not used in the amqp `pipelineConfigConsumer.ts`. Therefore I would remove this property for now and if we want to support thin events, we can change it in another PR.

Reply:
Well, there is no right and wrong here. Currently I tend to make it as easy as possible and agree on one. We can optimize later when we have a use case for it :)

Premature optimization is the mother of all evil they say ;)

But not sure if that is the right choice ...

Reply:
I happily removed it :) It is still present in the pipelineExecutionRequest, however. I did not remove it there because I want to be sure not to introduce another error (that is unrelated to the purpose of this pr). I create another issue for that.